### ISSUE TITLE
[feature] alternative design/style for message history

### ISSUE THREAD

**Issue by tetraf** [qTox/qTox#1310]
A config option to choose design of message history would be great.
With an IRC style (`[clock] <nick> text`) or a smartphone conversation style (with cartoon bubbles) or anything else (customisable?) according to the preferences of people.

**Comment by nshopik** [comment]
Yeah I'd prefer bubble style if dialog one-one no need to waste space on names, seems logical. I like how telegram on desktop done that (it also use Qt)

**Comment by ghost** [comment]
I'd really love having the IRC-style too.

**Comment by ProMcTagonist** [comment]
Could be done entirely with CSS chatform themes. We could ship with a couple, like SMS bubbles and IRC.

**Comment by ghost** [comment]
That sounds great!
